mis_builder/models/mis_report.py

```diff
@@ -354,7 +353,8 @@ def name_search(self, name="", args=None, operator="ilike", limit=100):
             ]
         )
         domain = osv_expression.AND([domain, name_search_domain])
-        return self.search(domain, limit=limit).name_get()
+        records = self.search(domain, limit=limit, order=self._order)
+        return [(record.id, record.display_name) for record in records.sudo()]
```

Comment:
Sorry but this is still not ok. You must override `_name_search`, not `name_search`, and return `self._search(...)`. There are plenty of examples in the Odoo source code.

Reply:
Done